### DISCUSSION TITLE
(ui) Ability to hide "scores" column

### DISCUSSION THREAD

**Discussion by marctorsoc** [langfuse/langfuse#3215] [Ideas]
### Describe the feature or potential improvement

For some reason, the column "scores" cannot be hidden in the Traces UI, in contrast to all other columns. There's also the ID column that cannot be hidden, but that I could accept. If I'm not logging any scores, why I cannot hide such column? 

Happy to work on this if approved

**Comment by marcklingen** [comment]
thanks for reporting, added an issue to track this: https://github.com/langfuse/langfuse/issues/3231

**Comment by marctorsoc** [comment]
this is now solved 🎉
